app/views/ems_infra/_recent-vms-line-chart.html.haml

```diff
@@ -0,0 +1,9 @@
+%div{'ng-controller' => "recentVmsLineChartController as vm", :class => "row", :style => "display:inline-block; width: 100%;"}
+  %div{:class => "col-md-6"}
+    %h3
+      {{vm.data.config.title}}
+    .row.row-tile-pf
+      .col-xs-6.col-sm-6.col-md-6.example-trend-container-vms
+        %pf-line-chart{:id => "vm.id", 'config' => "vm.config", 'chart-data' => "vm.data", 'set-area-chart' => "vm.custAreaChart", 'show-x-axis' => "vm.custShowXAxis", 'show-y-axis' => "vm.custShowYAxis"}
```

Comment:
same here, this just creates an id of `vm.id`

Reply:
same as below, `'linechart' + vm.id`